ts/Series/Lollipop/LollipopSeries.ts

```diff
@@ -54,7 +66,7 @@ const {
  * @augments Highcharts.Series
  *
  */
-class LollipopSeries extends DumbbellSeries {
+class LollipopSeries extends ScatterSeries {
```

Comment:
Although the Lollipop resembles a scatter series, it is more related to the Series. For example, the scatter has `jitter`, which does not make sense in the Lollipop. It works, but does not make sense. I suggest we extend Series directly, and by setting `lineWidth` to 0 by default (as it currently is), it looks like intended. It is also a simpler structure to not go via the ScatterSeries when we are not actually benefiting from it.

Reply:
Okey, got it! Thank you for the amazing overall clarification!